### PR TITLE
feat (refs T31375): use env var to connect to rabbitmq

### PR DESCRIPTION
**Pull request by mussbach**
**Ticket:** https://yaits.demos-deutschland.de/T31375

Rabbitmq parameters are consolidated into `rabbitmq_dsn` to be able to use a single environment variable `RABBITMQ_URL` to define the rabitmq dsn.

### How to review/test
Define a valid `RABBITMQ_URL` env var in your `.env.local` and e.g. import a docx file

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

**Review by breierDemos — APPROVED** [review]
lgtm

File: demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml [hunk 1]
```diff
@@ -40,7 +40,6 @@ parameters:
         - '*.demos-europe.eu'
         - 'cdnjs.cloudflare.com'
 
-    rabbitmq_host: "mq"
```
Comment (graupnerdemos) [inline demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml:43]: Shouldn't we set the `rabbitmq_dsn` parameter for the container setup in this file now?

**Review by graupnerdemos — CHANGES_REQUESTED** [review]
I warmly welcome the overlords of the dsn service configuration future. One question though.

File: demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml (hunk 1 above)
Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml:43]: No, we cannot, as this file is checked in into the public git

Comment (graupnerdemos) [reply demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml:43]: That reasoning doesn't exactly match with database, elasticsearch and mailhog being configured here too. 🤔

Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml:43]: But they are configured without users an passwords. Rabbitmq dns would contain both

Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml:43]: the variable `rabbitmq_host` does not exist any more

Comment (graupnerdemos) [reply demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml:43]: Well. We need another seamless solution soon then I guess.

**Review by graupnerdemos — APPROVED** [review]

**Pull request merged**